platforms/defaults_windows_test.go

```diff
@@ -47,15 +47,39 @@ func TestDefault(t *testing.T) {
 	}
 }
 
-func TestMatchComparerMatch(t *testing.T) {
+func TestDefaultMatchComparer(t *testing.T) {
+	defaultMatcher := Default()
+
+	for _, test := range []struct {
+		platform imagespec.Platform
+		match    bool
+	}{
+		{
+			platform: DefaultSpec(),
+			match:    true,
+		},
+		{
+			platform: imagespec.Platform{
+				OS: "linux",
+				Architecture: runtime.GOARCH,
```

Comment:
This would fail if ever run on x86 machine. Do we care?

Reply:
this should be updated if we dont' remove it based on https://github.com/containerd/containerd/pull/6491/files#r797781340